imaginairy/modules/refiners_sd.py

```diff
@@ -32,14 +32,17 @@
 
 
 def _tile_mode_conv2d_conv_forward(
-    self, input: torch.Tensor, weight: torch.Tensor, bias: torch.Tensor  # noqa
+    self,
+    tensor_input: torch.Tensor,
```

Comment:
keep naming

Reply:
done, changed it so noqa is on the same line as "input"